.github/workflows/merge-commits-to-next-branch.yml

```diff
@@ -0,0 +1,107 @@
+name: Merge commits to next branch
+
+on:
+  schedule:
+    - cron: "0 */2 * * *"
+
+env:
+  COMMIT_INFO: commit-info
+
+jobs:
+  dequeue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Azure Storage
+        run: npm i @azure/storage-queue
+      - name: Check queue first item
+        uses: actions/github-script@v6
+        with:
+          script: |
+              const { QueueClient, QueueServiceClient } = require("@azure/storage-queue");
+              const connectionString = `${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}`;
```

Comment:
Let me know if you need help with the secrets. I don't know what permissions are needed to add them.